src/array/kernel.cc

```diff
@@ -167,6 +166,15 @@ void SDDMMHetero(const std::string& op,
   });
 }
 
+int get_hs_id(HeteroGraphPtr graph, int target, dgl_type_t etype) {
```

Comment:
Docstring?  The name should also be more informative as well.

Also I would recommend wrapping it in an anonymous namespace.